app/src/main/java/com/codesoom/assignment/product/application/ProductService.java

```diff
@@ -0,0 +1,79 @@
+package com.codesoom.assignment.product.application;
+
+import com.codesoom.assignment.product.domain.Product;
+import com.codesoom.assignment.product.infra.ProductRepository;
+import com.codesoom.assignment.product.ui.dto.ProductResponseDto;
+import com.codesoom.assignment.product.ui.dto.ProductSaveRequestDto;
+import com.codesoom.assignment.product.ui.dto.ProductUpdateRequestDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * 상품 정보을 다룬다.
+ */
+@Transactional(readOnly = true)
+@Service
+@RequiredArgsConstructor
+public class ProductService {
+    private final ProductRepository productRepository;
+
+    /**
+     * 등록된 모든 상품 목록을 가져온다.
+     */
+    public List<ProductResponseDto> getProducts() {
+        return productRepository.findAll().stream()
+                .map(ProductResponseDto::of)
+                .collect(Collectors.toList());
```

Comment:
이쯤 됐으면 객체지향 생활체조의 `.`과 관련된 체조를 적용해 봐도 좋겠습니다.
한 줄에 점 하나만 찍기!
```suggestion
        return productRepository.findAll()
                .stream()
                .map(ProductResponseDto::of)
                .collect(Collectors.toList());
```

Reply:
평소 stream을 이후에 개행하고 있는데 객체지향 체조원칙을 이곳에도 적용할 수 있군요!